docker/Makefile

```diff
@@ -47,15 +43,13 @@ ifdef no_proxy
    DO_PROXY = 1
 endif
 ifdef DO_PROXY
-   DOCKER_COMPOSE_FILES_STL += sawtooth-pdo.proxy.yaml
    DOCKER_COMPOSE_FILES_CCF += ccf-pdo.proxy.yaml
    DOCKER_COMPOSE_FILES_CCF_ONLY += ccf.proxy.yaml
 endif
 ifdef PDO_DEBUG_BUILD
-   DOCKER_COMPOSE_FILES_STL += sawtooth-pdo.debugging.yaml
+   DOCKER_COMPOSE_FILES_CCF += ccf-pdo.debugging.yaml
 endif
```

Comment:
not sure i understand the point of this
probably a separate issue but the pdo build should have nothing to do with the ledger. and while there may be a need for the ccf debugging, that isn't really in the scope of this PR

Reply:
Cleaned up